llvm/lib/CodeGen/MIRParser/MIParser.cpp

```diff
@@ -1476,7 +1476,8 @@ bool MIParser::parseInstruction(unsigned &OpCode, unsigned &Flags) {
          Token.is(MIToken::kw_noconvergent) ||
          Token.is(MIToken::kw_unpredictable) ||
          Token.is(MIToken::kw_nneg) ||
-         Token.is(MIToken::kw_disjoint)) {
+         Token.is(MIToken::kw_disjoint) ||
+         Token.is(MIToken::kw_samesign)) {
```

Comment:
Missing MIRParser test

Reply:
Can you give me a pointer? The last one only had the import test:
https://github.com/llvm/llvm-project/pull/86650/files

Reply:
llvm/test/CodeGen/MIR/X86/fastmath.mir, should expand this to cover all the mir flags